Client/Frontend/Notifications/NotificationRootViewController.swift

```diff
@@ -160,11 +167,18 @@ private extension NotificationRootViewController {
     }
 
     @objc func didFinishSyncing() {
+        guard showingNotification else { return }
+
         showingNotification = false
         dispatch_async(dispatch_get_main_queue()) {
             self.hideStatusNotification()
         }
     }
+
+    @objc func fxaAccountDidChange() {
```

Comment:
`NotificationFirefoxAccountChanged` also gets fired when we remove an FxAAccount. In this case it doesn't indicate a first time sync. We should check that and behave accordingly